package sec2;

public class UserException1 {
	public static void main(String[] args) {
		method1();
	}
	public static void method1() {
		throw new RunException("사용자 예외");
	}
}